I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/variables.rpy", line 5, in script
    define advmc = Character([mcname], kind=adv)
  File "game/variables.rpy", line 5, in <module>
    define advmc = Character([mcname], kind=adv)
NameError: name 'mcname' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/variables.rpy", line 5, in script
    define advmc = Character([mcname], kind=adv)
  File "C:\Users\J\Documents\renpy-7.1.3-sdk\renpy\ast.py", line 1979, in execute
    value = renpy.python.py_eval_bytecode(self.code.bytecode)
  File "C:\Users\J\Documents\renpy-7.1.3-sdk\renpy\python.py", line 1937, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/variables.rpy", line 5, in <module>
    define advmc = Character([mcname], kind=adv)
NameError: name 'mcname' is not defined

Windows-8-6.2.9200
Ren'Py 7.1.3.1092
SpIn 1.0
Sat Mar 02 21:58:57 2019
